coalib/misc/Constants.py

```diff
@@ -109,7 +109,14 @@
                                     'languages',
                                     'definitions')
 
-system_coafile = os.path.join(coalib_root, 'default_coafile')
+# Checks for default_coafile in coala/coalib
+for root, dirs, file_lists in os.walk('coalib'):
+    for file in file_lists:
+        if file == 'default_coafile':
+            logging.warning
```

Comment:
Also `import logging` . Sorry @sils didn't know about the previous progress going on with this issue. lohani2280 informed me in gitter channel about it.